packages/tupaia-web/src/features/Map/MapOverlaysLayer/PolygonLayer.tsx

```diff
@@ -36,13 +35,10 @@ const ActivePolygon = styled(Polygon)<PolygonProps>`
 
 const ShadedPolygon = styled(BasePolygon)<PolygonProps>`
   fill: ${({ $shade }) => $shade};
-  stroke: ${({ $shade, $active }) => ($active ? POLYGON_HIGHLIGHT : $shade)};
+  stroke: ${({ $shade }) => $shade};
   fill-opacity: 0.5;
   &:hover {
-    fill-opacity: ${({ $active }) =>
-      $active
-        ? 0.5
-        : 0.8}; // don't add hover effect when it is the active polygon because it isn't clickable
```

Comment:
I actually think I remember why this was there - sometimes the active polygon would need to display as a shaded polygon, which it does, from the `getDisplayType` method. But in this case, the active polygon shouldn't be clickable, so removing the hover effect makes sense. Depends if you mind that an unclickable entity has a hover effect or not though?

Reply:
oh yeah that's a good idea, I'll put the change back. I should have read the comment more carefully :)